react/components/ErrorBoundary.tsx

```diff
@@ -7,32 +7,33 @@ interface Props {
 }
 
 class ErrorBoundary extends React.Component<Props> {
-  state = {
+  private _state = {
     error: undefined,
     errorInfo: undefined,
   }
 
-  componentDidCatch(error: Error, errorInfo: ErrorInfo) {
+  public get state() {
```

Comment:
Since it's React API, maybe it isn't a good idea to shouldn't override `this.state`